src/Tabs/Tabs.stories.mdx

```diff
@@ -0,0 +1,89 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import { withKnobs, boolean, select } from '@storybook/addon-knobs';
+import { Tabs } from './Tabs';
+import { Badge } from '../Badge'
+
+<Meta title="Components|Tabs" component={Tabs} />
+
+# Button
+
+## Description
+
+Tabs component
+
+## Preview
+
+<Preview>
+  <Story name="Default tabs with three options">
+    <Tabs initialTab="tab-2">
+      <Tabs.TabWrapper>
+        <Tabs.Tab tab="tab-1">Tab A</Tabs.Tab>
+        <Tabs.Tab tab="tab-2">Tab B</Tabs.Tab>
+        <Tabs.Tab tab="tab-3">Tab c</Tabs.Tab>
+      </Tabs.TabWrapper>
+      <Tabs.Panel whenActive="tab-1">a panel</Tabs.Panel>
+      <Tabs.Panel whenActive="tab-2">b panel</Tabs.Panel>
+      <Tabs.Panel whenActive="tab-3">c panel</Tabs.Panel>
+    </Tabs>
+  </Story>
+</Preview>
+
+<Preview>
+  <Story name="Three tabs strached">
```

Comment:
spelling

Reply:
When you see a spelling mistake can you let me know which word it is? I know it should be obvious, but really not easy for me.

Reply:
Strached should be stretched

Reply:
aw yeah, I got there eventually, just took me while.